### DISCUSSION TITLE
Number of unique peptides resulting in identification

### DISCUSSION THREAD

**Discussion by SauerTho** [vdemichev/DiaNN#171] [Q&A] [answered]
Dear all,

I am looking for a setting or  the information in the reports, on how many peptides contributed to a specific protein in terms of identification and quantification. Is there a command to exclude single hit proteins? Maybe also in the corresponding R Package handling the DIA-NN reports?

Best
Thorben

**Comment by vdemichev** [comment, marked as answer]
Hi Thorben,

You can count the number of peptides in R/Python, using the main report produced by DIA-NN.
The code is sth like this:
x <- unique(df[df$Global.Q.Value <= 0.01,c('Protein.Group','Stripped.Sequence')])
n_peps <- table(df$Protein.Group)
Add can exclude single hit proteins based on this.

Best,
Vadim
